sql/src/main/java/io/crate/analyze/ShowStatementAnalyzer.java

```diff
@@ -74,10 +82,16 @@ Query rewriteShowTransaction() {
     }
 
     AnalyzedStatement analyzeShowTransaction(Analysis analysis) {
-        Query query = rewriteShowTransaction();
-        Analysis newAnalysis = analyzer.boundAnalyze(query, analysis.transactionContext(), ParameterContext.EMPTY);
-        analysis.rootRelation(newAnalysis.rootRelation());
-        return newAnalysis.analyzedStatement();
+        return unboundAnalyze(rewriteShowTransaction(), analysis, ParamTypeHints.EMPTY);
```

Comment:
why not use `unboundAnalyze(query, analysis)` and thus use `analysis.paramTypeHints()` here  as well?

Reply:
The original method passed explicitly `ParamTypeHints.EMPTY` but I just see while debugging through that the analysis has anyway empty parameters, so this makes totally sense.